src/set_relation/set_relation.cc

```diff
@@ -2938,9 +2938,9 @@ void VisitorBoundDomainRange::preVisitUFCallTerm(UFCallTerm * t){
     // Determine what the output arity of this particular UF call is
     // taking into consideration that it could be indexed.
     unsigned int out_arity = range->arity();
-    if (uf_call->isIndexed()) {
-        out_arity = 1; // only one element is being accessed
-    } 
+//    if (uf_call->isIndexed()) {
```

Comment:
I understand what this means semantically. However, it has no positive use
in practice. at least non that I could find. And in the implementation,
this is actually causing one the test cases to fail
(NormalizationTest.buildingNormalize).

On Thu, Jun 23, 2016 at 8:50 PM, Michelle Strout <notifications@github.com>
wrote:

> In src/set_relation/set_relation.cc
> <https://github.com/CompOpt4Apps/IEGenLib/pull/38#discussion_r68265958>:
>
> > @@ -2938,9 +2938,9 @@ void VisitorBoundDomainRange::preVisitUFCallTerm(UFCallTerm * t){
> >      // Determine what the output arity of this particular UF call is
> >      // taking into consideration that it could be indexed.
> >      unsigned int out_arity = range->arity();
> > -    if (uf_call->isIndexed()) {
> > -        out_arity = 1; // only one element is being accessed
> > -    }
> > +//    if (uf_call->isIndexed()) {
>
> Can we delete these lines?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CompOpt4Apps/IEGenLib/pull/38/files/873a894609e6a1eb8305ba539144baac69001790#r68265958>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ANGVl5GLp3bJuSA5DZwc_VVxYcFtn6Nzks5qOrJNgaJpZM4I89jG>
> .
>